Source/gmenu.cpp

```diff
@@ -415,11 +415,10 @@ void gmenu_slider_1(TMenuItem *pItem, int min, int max, int gamma)
 
 int gmenu_slider_get(TMenuItem *pItem, int min, int max)
 {
-	int v3;          // eax
-	unsigned int v4; // ecx
+	int v3, v4;
 
-	v3 = (pItem->dwFlags >> 12) & 0xFFF;
 	v4 = pItem->dwFlags & 0xFFF;
+	v3 = ((signed int)pItem->dwFlags >> 12) & 0xFFF;

```

Comment:
probably dwFlags should be a DWORD seeing as its called dw*

Reply:
dunno, without this cast it's not bin exact, dwFlags is DWORD by default

Reply:
if you want, I can get rid of it by splitting this into 3 lines 🤷‍♂